python/lsst/ts/ofc/scripts/generate_sensitivity_matrix.py

```diff
@@ -0,0 +1,362 @@
+import argparse
+from astropy.io import fits
+import batoid
```

Comment:
I suspect the intention here is more "document where the sensitivity matrix came from" as opposed to "here's a thing we expect users to run"?  If so, then I don't think we need to have ts_ofc depend on batoid or galsim.  Maybe this file needs to live in a different part of the repo then though?

Reply:
Just to add to this discussion, galsim will be needed regardless because we use it to evaluate the double zernike object. I think batoid might not be needed. I defer to Tiago to where it should be placed if not here